README.md

```diff
@@ -1,2 +1,210 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo and presentation video](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+## 📖 practice-react <a name="about-project"></a>
+
+<h5> I set up the environment and tools needed to develop a React application. In this project, I have made a simple SPA To-Do-App project that allows user to add and remove To Do and display on UI .</h5>
+
+## 🛠 Built With <a name="built-with"></a>
+
+## Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <b>This project only works client side right now.</b>
+  <ul>
+    <li>JSX</li>
+    <li>CSS</li>
+    <li>JS</li>
+    <li>React</li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href=#>Local Server - It will be Hosted on Github when finish</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href=#>I will us API's in future</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+## Key Features <a name="key-features"></a>
+
+- Add To-Do Items
+- Remove To-Do Items from list
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo<a name="live-demo"></a>
+
+- [Live Demo Link](https://feature-2--sparkling-dieffenbachia-24a95b.netlify.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+## Prerequisites <a name="prerequisites"></a>
+
+- In order to run this project you need:
+
+### It would be best if you had some familiarity with `JS`, `CSS`, and `React` .
+
+- A Computer (MAC or PC)
+- code editor (VSCode,Atom etc...)
+- A browser (Chrome,Mozilla,Safari etc...)
+- Version Control System (Git and Github)
+
+# Setup <a name="setup"></a>
+
+Clone this repository to your desired folder:
+
+```bash
+       git clone https://github.com/AbdusSattar-70/practice-react.git
+       cd practice-react
+```
+
+# Install <a name="install"></a>
+
+Install this project with:
+
+```bash
+     npm install
+```
+
+# Run tests and check errors <a name="run-tests"></a>
+
+To run tests and check errors, run the following command:
+
+- To check Styelint error:-
+
+```bash
+   npm test
+```
+
+```bash
+   npx stylelint "\*_/_.{css,scss}"
+```
+
+- To check Eslint error:-
+
+```bash
+  npx exlint .
+```
```

Comment:
please make sure that you are using the correct eslint checker. to clarify the issue u can refer this [linters](https://github.com/microverseinc/linters-config/tree/master/react-redux)

Reply:
Hello @Santosh-Konappanavar,
Thanks. I forgot to mention the react linter checker command. I will mention the right command.